### DISCUSSION TITLE
Mongo 6.0 Existing Database woes

### DISCUSSION THREAD

**Discussion by csellis** [prisma/prisma#16384] [Q&A] [answered]
Referencing #16363 /bin/sh: mongodb: command not found

### Bug description

When following the Prisma YT connection tutorials, I'm getting `/bin/sh: mongodb: command not found` when trying to connect to a remote repository. 

I've done a `npx prisma db pull` and have successfully created my schema.prisma file. This indicates the correct `.env` import is coming in.

I have tried upgrading to Mongo 6.0, so my ~/.zshrc file now looks like:
```
export PATH=/opt/homebrew/Cellar/mongodb-community/6.0.1/bin:$PATH
```
pwd inside that bin
```
christopherellis:bin/ (stable) $ pwd                                 [17:49:21]
/opt/homebrew/Cellar/mongodb-community/6.0.1/bin
christopherellis:bin/ (stable) $ ls                                  [17:49:24]
install_compass mongod          mongos
christopherellis:bin/ (stable) $   
```
`mongod` and `mongosh` both work. I don't see MongoDB command in any of the bins

Error
```
mac:apply-sveltekit/ (main✗) $ npx prisma generate                                                                                            [7:27:09]
Environment variables loaded from .env
Prisma schema loaded from prisma/schema.prisma
Error: Generator at mongodb could not start:

/bin/sh: mongodb: command not found
```

### How to reproduce

1. Follow instructions here [https://www.youtube.com/watch?v=BwVOIRX3VXk&ab_channel=Prisma](https://www.youtube.com/watch?v=BwVOIRX3VXk&ab_channel=Prisma)
2. Install homebrew
3. Install mongo-community@4.4 as described
4. Add to zsh `export PATH=/opt/homebrew/Cellar/mongodb-community@4.4.16/bin:$PATH`
6. set `.env` 
```
DATABASE_URL="mongodb+srv://<usr>:<password>@sumfin.onMongodb.mongodb.net/TABLE"
```

### Expected behavior

I expect it would generate my Prisma types

### Prisma information

```
generator client {
  provider = "mongodb"
}

datasource db {
  provider = "mongodb"
  url      = env("DATABASE_URL")
}
```

### Environment & setup

- OS: M1 MacOS
- Database: MongoDB hosted
- Node.js version: Now using node v16.18.0 (npm v9.1.2)

### Prisma Version

```
using npx
```

**Comment by janpio** [comment, marked as answer]
Solution: https://github.com/prisma/prisma/issues/16363#issuecomment-1322844601

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
